drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -2420,6 +2420,11 @@ matterManufacturer:
     vendorId: 0x138A
     productId: 0x1392
     deviceProfileName: matter-bridge
+  - id: "4932/1054"
+    deviceLabel: EUD64NPN-IPM
```

Comment:
Every other matter bridge has the device label "Matter Bridge". I think we should maintain this if possible. The current label `EUD64NPN-IPM` seems confusing to me, so if the partner is okay with us changing it I would vouch for a name that is more informative.